libbeat/cmd/instance/beat.go

```diff
@@ -1041,7 +1041,7 @@ func LoadKeystore(cfg *common.Config, name string) (keystore.Keystore, error) {
 	return keystore.Factory(keystoreCfg, defaultPathConfig)
 }
 
-func initKibanaConfig(beatConfig beatConfig) (*common.Config, error) {
+func InitKibanaConfig(beatConfig beatConfig) (*common.Config, error) {
```

Comment:
This function never returns an error. I suggest changing the function signature to remove the error so that error handling can be removed where it's used.

Reply:
Good catch! I will remove the return error! Thanks!